chain-signatures/node/src/indexer.rs

```diff
@@ -53,6 +54,14 @@ pub struct Options {
     /// The threshold in seconds to check if the indexer needs to be restarted due to it stalling.
     #[clap(long, env("MPC_INDEXER_RUNNING_THRESHOLD"), default_value = "300")]
     pub running_threshold: u64,
+
+    /// The threshold in block height lag to check if the indexer has caught up.
```

Comment:
Let's discuss a strategy for configuration at our next meeting. Do we want to put everything in the contract?

Reply:
yeah, we should move all these indexer configurations into the contract to make it easier to configure

Reply:
Yeah I was thinking the same when adding timeout options today.